src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -179,6 +179,16 @@ public static class Captions
       public static readonly string Exception = "Exception";
       public static readonly string StackTrace = "Stack Trace";
 
+      public static string ShouldWatermarkBeUsedForChartExportToClipboard(string applicationName, string optionLocation)
```

Comment:
Text shown to user upon application start. Because this question is clearly not something that people would expect, it needs to be very clear as to WHAT this is for.